_posts/2018-05-09-Stop-_-Frisk.md

```diff
@@ -26,21 +26,21 @@ The analysis also provides a number of interactive visualizations that can allow
+A stop and frisk, or Terry stop, are investigatory detentions that police may initiate based on reasonable suspicion that the detained individual is engaged, or about to engage, in criminal conduct. In addition, police may conduct a limited search for weapons in order to protect the officer or public safety when the officer has reasonable suspicion that a stopped individual is “armed and dangerous.” While police may only initiate a frisk to search for weapons, they may seize contraband discovered during the frisk. In both cases, the standard of reasonable suspicion means that police must point to specific and articulable facts, although a set of individually innocuous facts may meet the standard for reasonable suspicion.
 
-Named for the Supreme Court case that defined this authority, Terry v. Ohio, 392 U.S. 1 (1968), Terry stops rose to prominence through controversy over racial profiling in the NYPD’s “stop, question, and frisk” policing strategy. At its height in 2011, the NYPD made 685,724 stops; the vast majority, over 85%, of these stopped were African American or Latino. Proponents argue that it preempts crime by allowing officers to intervene and check for illegal weapons that may be used in future crimes, while opponents argue that the law allows police officers to harass people of color with weak, or non-existent, reasonable suspicion of criminal conduct. [Research suggests that the NYPD’s stops had few effects on robbery and burglary.](https://doi.org/10.1080/07418825.2012.712152)
+Named for the Supreme Court case that defined this authority, Terry v. Ohio, 392 U.S. 1 (1968), Terry stops rose to prominence through controversy over racial profiling in the NYPD’s “stop, question, and frisk” policing strategy. At its height in 2011, the NYPD made 685,724 stops; over 85% of those stopped were African American or Latino. Proponents argue that it preempts crime by allowing officers to intervene and check for illegal weapons that may be used in future crimes, while opponents argue that the law allows police officers to harass people of color with weak or non-existent suspicion of criminal conduct. [Research suggests that the NYPD’s stops had few effects on robbery and burglary.](https://doi.org/10.1080/07418825.2012.712152) <- I WOULD TAKE OUT THIS SENTENCE/LINK. In the abstract it says: "Contrary to prior research, the study reveals few effects of SQF on robbery and burglary. We caution against drawing definitive conclusions from both the current and prior research and recommend that future research address both the impact of SQF on crime and possible collateral effects on the rights and liberties of citizens in the communities most affected by the policy." Makes it seem biased to include this study without acknowledging the others, but I don't think you need to go into that analysis. 
```

Comment:
GW: Rather than remove the sentence, I changed to mention the inconclusive findings from previous research, which is still valuable to have, in my opinion.